src/Microsoft.AspNetCore.Mvc.Core/Internal/ControllerActionInvokerCacheEntry.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Mvc.Filters;
+using Microsoft.Extensions.Internal;
+
+namespace Microsoft.AspNetCore.Mvc.Internal
+{
+    public class ControllerActionInvokerCacheEntry
+    {
+        internal ControllerActionInvokerCacheEntry(
+            FilterItem[] filters,
+            Func<ControllerContext, object> controllerFactory,
+            Action<ControllerContext, object> controllerReleaser,
+            ControllerBinderDelegate controllerBinderDelegate,
+            ObjectMethodExecutor actionMethodExecutor)
+        {
+            ControllerFactory = controllerFactory;
+            ControllerReleaser = controllerReleaser;
+            ControllerBinderDelegate = controllerBinderDelegate;
+            Filters = filters;
```

Comment:
Can you call this `cachedFilters` or something to make it clear that it's not supposed to be used as-is?